pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>de.brandwatch.minianalytics</groupId>
+  <artifactId>mini-analytics</artifactId>
+  <version>0.1</version>
+  <modules>
+    <module>mention-storer</module>
+    <module>mentions-generator</module>
+  </modules>
+  <packaging>pom</packaging>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-autoconfigure</artifactId>
+      <version>2.1.7.RELEASE</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.data</groupId>
+      <artifactId>spring-data-solr</artifactId>
```

Comment:
I doubt that all your apps depend on Solr, so there is no need to put it into your parent pom. This might be true for other apps as well. I recommend to use the parent pom mainly for [Dependency Management](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management), meaning you just define the versions of things here within a <dependencyManagement> tag. You might additionally actually globally define dependencies that are truly global (i.e. required by every module), but those are usually not many. Everything else you can specify in the app specific pom files without a version (as that's defined in the dependency management of the parent pom already). That way you manage your versions in one place but still only have the dependencies you need.